libs/design/sidebar/examples/src/basic-sidebar/basic-sidebar.component.html

```diff
@@ -1,11 +1,8 @@
 <daff-sidebar-viewport>
-  <daff-sidebar>
-    <a daff-link-set-item routerLink="accordion">Accordion</a><br>
-    <a daff-link-set-item routerLink="article">Article</a><br>
+  <daff-sidebar class="basic-sidebar">
+    Sidebar content
   </daff-sidebar>
-  <div>
-    <nav daff-navbar>
-      <div>Basic Sidebar</div>
-    </nav>
+  <div class="basic-sidebar__content">
```

Comment:
Why did the content change?

Reply:
This is just the example? I just condensed and cleaned it up.